lib/components/potentiometer.ts

```diff
@@ -0,0 +1,22 @@
+import { resistance } from "circuit-json"
+import {
+  type CommonComponentProps,
+  commonComponentProps,
+  lrPins,
+} from "lib/common/layout"
+import { expectTypesMatch } from "lib/typecheck"
+import { z } from "zod"
+
+export interface PotentiometerProps extends CommonComponentProps {
+  resistance: number | string
+  wiper?: number
+}
+
+export const potentiometerProps = commonComponentProps.extend({
+  resistance,
+  wiper: z.number().min(0).max(1).optional().default(0.5),
```

Comment:
hmm doesn't make sense to have the wiper here since we aren't doing simulation yet, I think it's also kind of unclear what it does. Probably best to do just do `maxResistance` for now

https://github.com/tscircuit/jlcsearch/blob/a09a6d3f554ef18bfdcbf27efcc005cbbc6d2e11/docs/generated/resistors.md?plain=1#L46

Reply:
note: `maxResistance` should be the only property. Potentiometers have a variable resistance based on the wiper position. It follows a curve that is typically logarithmic but sometimes linear for a linear pot

![image](https://github.com/user-attachments/assets/6458eb2c-030a-498e-9d78-fcc5f5c36177)